hips/tiles/tests/test_surveys.py

```diff
@@ -85,8 +96,6 @@ def test_fetch(self):
         surveys = HipsSurveyPropertiesList.fetch()
         assert len(surveys.data) > 3
 
-        # TODO: look up survey by name here
-        # Otherwise this will break when the list changes
-        survey = surveys.data[0]
+        survey = HipsSurveyProperties.from_name('CDS/P/2MASS/H')
```

Comment:
This change to the test is no good.

I would suggest to move the code that does the name lookup from the `HipsSurveyProperties.from_name` method to a `HipsSurveyProperties.from_name` method and then call this here from this test (instead of calling
```
HipsSurveyProperties.from_name('CDS/P/2MASS/H')
```
which will fetch the list again.

The implementation of `HipsSurveyProperties.from_name` would then just be
```python
@classmethod
def from_name(name);
    surveys = HipsSurveyProperties.fetch()
    return surveys.from_name(name)
```
and maybe we'd remove it eventually.

You should also add a test with an invalid key like `name='kronka lonka'` and check that a KeyError is raised.
I explained how to do that with `pytest.raises` here previously: https://github.com/hipspy/hips/pull/55#discussion_r125573377